API/64/user-posts-comments.js

```diff
@@ -0,0 +1,83 @@
+const apiCall = async (userID) => {
+  try {
+    const response = await fetch(
+      `https://jsonplaceholder.typicode.com/posts?userId=${userID}`,
+      {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    );
+    const posts = await response.json();
```

Comment:
Can we take these repetitions out and write a function which abstract the fetch api call?